Core.lua

```diff
@@ -222,10 +222,17 @@ function WarpDeplete:HideBlizzardObjectiveTracker()
     end
 
     self:PrintDebug("Re-hiding objective frame")
-    ObjectiveTrackerFrame:Hide()
+    self:CheckAndHideObjectiveTrackerFrame()
   end)
 end
 
+-- Why repeat myself :)
+function WarpDeplete:CheckAndHideObjectiveTrackerFrame()
+  if ObjectiveTrackerFrame:IsVisible() then
+    ObjectiveTrackerFrame:Hide()
+  end
+end
+
```

Comment:
```suggestion
function WarpDeplete:HideObjectiveTrackerFrame()
  if ObjectiveTrackerFrame:IsVisible() then
    ObjectiveTrackerFrame:Hide()
  end
end

```

The comment is not needed, I think. Also, I'm fine with the name not including `CheckAnd`, since the end result of this is always that the frame will be hidden if it isn't yet - the checking is just an optimization.

If we hide on every tick anyways, maybe we don't need to timer re-hide logic in `HideBlizzardObjectiveTracker()`. Maybe you can just replace the content of that function with this straight up?